api/app/src/main/kotlin/packit/repository/PacketRepository.kt

```diff
@@ -8,6 +8,6 @@ import packit.model.Packet
 @Repository
 interface PacketRepository : JpaRepository<Packet, String>
 {
-    @Query("select p.id from Packet p")
+    @Query("select p.id from Packet p order by p.id asc")
     fun findAllIds(): List<String>
```

Comment:
You surely don't need the Query. `findAllById` will return all Ids. `findAllByIdOrderByIdAsc` will return ordered list.I think you don't need to write anyCode inside the this repository.

Reply:
I guess it might not work because the Id is stored as String.

Reply:
You do need to do a little work to select only a single column, see e.g. https://stackoverflow.com/questions/22007341/spring-jpa-selecting-specific-columns. Using a specific query seemed like the neatest way to me (creating a custom object with only the id field just adds more boilerplate).

Reply:
You do need to do a little work to select only a single column, see e.g. https://stackoverflow.com/questions/22007341/spring-jpa-selecting-specific-columns. Using a specific query seemed like the neatest way to me (creating a custom object with only the id field just adds more boilerplate).